Bugzilla/API/1_0/Util.pm

```diff
@@ -17,7 +17,7 @@ use Bugzilla::Flag;
 use Bugzilla::FlagType;
 use Bugzilla::Util qw(datetime_from email_filter);
 
-use JSON;
+use JSON::MaybeXS;
```

Comment:
Also need to convert:

JSON::true => JSON->true
JSON::false => JSON->false
JSON::null => undef

otherwise we have a compile error of:

Bareword "JSON::true" not allowed while "strict subs" in use at Bugzilla/API/Server.pm line 152.